src/main/java/io/strimzi/test/container/StrimziKafkaContainer.java

```diff
@@ -104,15 +131,54 @@ protected void doStart() {
     }
 
     /**
-     * Fluent method, which sets @code{externalZookeeperConnect}
+     * Fluent method, which sets @code{externalZookeeperConnect}.
+     *
+     * If the broker was created using Kraft, this method throws an {@link IllegalArgumentException}.
+     *
      * @param connectString connect string
      * @return StrimziKafkaContainer instance
      */
     public StrimziKafkaContainer withExternalZookeeper(final String connectString) {
+        if (useKraft) {
+            throw new IllegalArgumentException("Cannot configure an external Zookeeper and use Kraft at the same time");
+        }
         this.externalZookeeperConnect = connectString;
         return self();
     }
 
+    /**
+     * Fluent method, which sets a waiting strategy to wait until the broker is ready.
+     *
+     * This method waits for a log message in the broker log.
+     * You can customize the strategy using {@link #waitingFor(WaitStrategy)}.
+     *
+     * @return StrimziKafkaContainer instance
+     */
+    public StrimziKafkaContainer withForRunning() {
```

Comment:
I guess this should be `waitForRunning`, or possibly `withWaitForRunning`.

Reply:
Oh.... I write the right version in the README and not in the code. Let me fix that (`waitForRunning()` it is.)

Reply:
Fixed.